src/transformers/models/realm/modeling_realm.py

```diff
@@ -1737,11 +1742,21 @@ def __init__(self, config, retriever=None):
         self.post_init()
 
     @property
-    def beam_size(self):
+    def searcher_beam_size(self):
         if self.training:
             return self.config.searcher_beam_size
         return self.config.reader_beam_size
 
+    def block_embedding_to(self, device):
```

Comment:
Ok for me even though I don't think it's really necessary

Reply:
@patrickvonplaten - Do you think if it is better that I just add some tips in the model docs, e.g. telling users they can manually send `block_emb` back to cpu if their gpu memory is not sufficient, and remove this function.

We still need to prompt users how they can fine-tune the model on a 12Gb memory GPU as promised by the paper.

Reply:
I think the best would be to add a comment under `Implementation details` on the model doc above this section: https://github.com/huggingface/transformers/blob/master/docs/source/model_doc/realm.mdx#realmconfig 

But it's also ok for me to leave as is now - your choice :-)